src/pages/index.js

```diff
@@ -118,31 +126,79 @@ function handleCloseModal(modal) {
 }
 
 function handleProfileFormSubmit(evt) {
-  evt.preventDefault();
-  handleCloseModal(profileFormElement);
+  function makeRequest() {
+    return api
+      .editUserInfo({ name: nameInput.value, about: jobInput.value })
+      .then((userData) => {
+        profileNameElement.textContent = userData.name;
+        profileJobElement.textContent = userData.about;
+        handleCloseModal(profileFormElement);
+      });
+  }
+  handleSubmit(makeRequest, evt);
 }
 
-function renderCards() {
-  initialCards.forEach((card) => {
-    cardsListElement.append(getCardElementData(card));
-  });
+function handleNewPostSubmit(evt) {
+  function makeRequest() {
+    return api.postNewCard(captionInput.value, linkInput.value).then((res) => {
+      const cardElement = getCardElementData(res);
+      cardsListElement.prepend(cardElement);
+      evt.target.reset();
+      disableButton(newPostSubmitButton, settings);
+      handleCloseModal(newPostFormElement);
+    });
+  }
+  handleSubmit(makeRequest, evt);
 }
 
-function renderNewCard(card) {
-  cardsListElement.prepend(getCardElementData(card));
+function handleAvatarSubmit(evt) {
+  function makeRequest() {
+    return api.editUserAvatar(avatarInput.value).then((data) => {
+      profileImg.src = data.avatar;
+      handleCloseModal(avatarModal);
+      evt.target.reset();
```

Comment:
`evt.target.reset();` is already inside `handleSubmit`, so you don't need it here anymore